install/runners/nunit-tools.wxs

```diff
@@ -105,11 +104,15 @@
                       ProcessorArchitecture="msil" 
                       Source="$(var.BuildDir)nunit.engine.dll" />
 			</Component>
-            <Component Id="NUNIT.ENGINE.ADDINS">
-                <File Id="nunit.engine.addins"
+			<Component Id="NUNIT.ENGINE.ADDINS">
+				<File Id="nunit.engine.addins"
                       Source="$(var.BuildDir)nunit.engine.addins" />
-            </Component>
-            <Component Id="NUNIT.ENGINE.API">
+			</Component>
```

Comment:
You and I need to get our editors set up consistently for XML, we both have different settings. I think I might even have different settings between a couple of the editors I use.

I propose tabs/indents at 2 spaces and replace tabs with spaces? 4 spaces would be fine too if you prefer that. Once we decide, I will do a quick reformat of a few key files in another PR.